docker/compose/docker-compose-services.yml

```diff
@@ -5,7 +5,7 @@ postgres:
   ports:
     - "5432:5432"
   volumes:
-    - ${DOCKER_DATA_HOME}/postgresql:/var/lib/postgresql/data
+    - /data/postgresql:/var/lib/postgresql/data
```

Comment:
I think this change would break current docker on linux (we already have data elsewhere, and I don't think /data is a thing for linux OS). Can you change the `DOCKER_DATA_HOME` initialization in docker/_include.sh instead?